packages/components/molecules/f-skeleton-loader/src/components/skeletons/Sentence.vue

```diff
@@ -1,24 +1,26 @@
 <template>
-    <div class="c-skeleton-loader-sentence c-skeleton-loader-bone">
-        <text-block />
-        <text-block />
+    <div>
+        <text-block :class="$style['c-sentence-textBlock']" />
```

Comment:
I'd be tempted to prefix all subcomponents with `c-skeletonLoader-blah`, so that they have the base component context in the classnames?

Reply:
Hey @ashleynolan happy to do this, just wanted to check what that would look like for - https://github.com/justeat/fozzie-components/pull/848#discussion_r616770421 

Should I only prefix the child components?